woocommerce/woocommerce-android
# Always show label about awaiting payments

Closes: #6780


### Description

Improvements on the order details screen regarding payment status (check the screenshots)

### Testing instructions

* Create an order via the app
* Notice unpaid state matches the image below 
* Pay for the order
* Notice unpaid state matches the image below 
* Create an order via the web (how via normal flow of buying something in a store as a customer)
* Notice unpaid state matches the image below 
* Pay for the order
* Notice unpaid state matches the image below 

### Images/gif


To align with the iOS implementation:

#### Paid order without payment title
<img width="442" alt="image" src="https://user-images.githubusercontent.com/4923871/174978511-ef1ff72e-39df-4ad4-a927-a08427486ec0.png">



#### Paid order with payment title
<img width="457" alt="image" src="https://user-images.githubusercontent.com/4923871/174978270-d01f29aa-69a4-4e5a-8e2c-be67dc02a747.png">


#### Unpaid order without payment title
<img width="449" alt="image" src="https://user-images.githubusercontent.com/4923871/174981205-1e5bd7d5-f2a2-4140-8c82-892ef3d3e865.png">



#### Unpaid order with payment title

<img width="463" alt="image" src="https://user-images.githubusercontent.com/4923871/174979721-6e8f3e21-7e83-4522-8f3a-2670186b0c73.png">
